components/network/memcached/Makefile

```diff
@@ -21,36 +21,51 @@
 
 #
 # Copyright (c) 2011, 2013, Oracle and/or its affiliates. All rights reserved.
+# Copyright (c) 2023, Friedrich Kink. All rights reserved.
 #
 
-BUILD_BITS=64
+BUILD_BITS=		64
+OPENSSL_VERSION=	3.1
+
```

Comment:
There doesn't seem to be a depedency on openssl at all.

Reply:
There would be if `#CONFIGURE_OPTIONS +=   --enable-tls` would be enabled. For the time being it does not work. So it is just a short cut/help/reminder in case being enabled again.

Reply:
How essential is TLS for memcached?

Reply:
Both `BUILD_BITS=64` and `OPENSSL_VERSION=3.1` are defaults.  Setting them here is a call for more work in future.  So please remove both.  Thank you.